frontend/public/locales/fi/translation.json

```diff
@@ -43,7 +43,13 @@
     "start_date": "Virkaan asettamispäivämäärä",
     "ending_date": "Virkasopimuksen päättämispäivämäärä",
     "no_end_date": "Ei määräaikaa",
-    "save": "Tallenna"
+    "save": "Tallenna",
+    "replacement": "Sijaisuus",
+    "replacement_active": "Sijaisuus voimassa",
+    "replacement_details": "Sijaisuuden tiedot",
+    "replacement_start_date": "Sijaisuuden alkamispäivämäärä",
+    "replacement_end_date": "Sijaisuuden päättymispäivämäärä",
```

Comment:
Not really related and can be easily fixed later, but just a note that I think every "...päivämäärä" should be just "...päivä". I.e. "start_date": "Virkaan asettamispäivämäärä", --> "Virkaan asettamispäivä" or "Sijaisuuden alkamispäivä"